packages/react-scripts/config/webpack.config.js

```diff
@@ -138,7 +156,10 @@ module.exports = function(webpackEnv) {
         {
           loader: require.resolve(preProcessor),
           options: {
-            sourceMap: true,
+            ...preProcessorOptions,
+            ...{
+              sourceMap: isEnvProduction && shouldUseSourceMap,
```

Comment:
Whats the purpose of this line? We added this so that we could customise preprocessor options, the `sourceMap` option should be set on invocation of this no? (it is: https://github.com/Skyscanner/backpack-react-scripts/pull/75/files#diff-dc0c4e7c623b73660da1809fc60cf6baR586)

Reply:
This was from BRS 7 - so not sure what its purpose was

Reply:
try using sourceMap: true

Reply:
Done and works 😄